sorc/chgres_cube.fd/program_setup.f90

```diff
@@ -98,16 +94,71 @@ module program_setup
+!                                 from the varmap file. 
 ! use_thomp_mp_climo              When true, read and process Thompson
 !                                 MP climatological tracers.  False,
 !                                 when 'thomp_mp_climo_file' is NULL.
 ! vcoord_file_target_grid         Vertical coordinate definition file
 ! wltsmc_input/target             Wilting point soil moisture content
 !                                 input/target grids
-! 
+!
+! nsoill_out                      Number of soil levels desired in the output data. 
+!                                 chgres_cube can interpolate from 9 input to 4 output
+!                                 levels. 
+!                                 DEFAULT: 4    
+!
+! Variables that are relevant only for "grib2" input type:
+!
+! grib2_file_input_grid           REQUIRED. File name of grib2 input data.
+!                                 Assumes atmospheric and surface data are in a single 
+!                                 file. 
+!
+! varmap_file                     REQUIRED. Full path of the relevant varmap file. 
+!
+! external_model                  The model that the input data is derived from. Current
+!                                 supported options are: "GFS", "HRRR", "NAM", "RAP". 
+!                                 Default: "GFS"
+!
+! vgtyp_from_climo                If false, interpolate vegetation type from the input 
+!                                 data to the target grid instead of using data from 
+!                                 static data. Use with caution as vegetation categories
+!                                 can vary. 
+!                                 Default: False
+!
+! sotyp_from_climo                If false, interpolate soil type from the input 
+!                                 data to the target grid instead of using data from 
+!                                 static data. Use with caution as the code assumes
+!                                 input soil type use STATSGO soil categories.
+!                                 Default: False
+!
+! vgfrc_from_climo                If false, interpolate vegetation fraction from the input 
+!                                 data to the target grid instead of using data from 
+!                                 static data. Use with caution as vegetation categories
+!                                 can vary. 
+!                                 Default: False
+!
+! minmax_vgfrc_from_climo         If false, interpolate min/max vegetation fraction from 
+!                                 the input data to the target grid instead of using data
+!                                 from static data. Use with caution as vegetation
+!                                 categories can vary. 
+!                                 Default: False
+!
+! lai_from_climo                  If false, interpolate leaf area index from the input 
```

Comment:
FYI.  We have a global, weekly 5km LAI climo dataset available.  It can be mapped to any grid - regional or global - as part of the grid generation process.

Reply:
@GeorgeGayno-NOAA We use 30 s MODIS LAI data available in WRF GEOG for our LAI in the HRRR, which is needed for our FV3-LAM to HRRR comparisons for RRFS readiness.  It would be good to provide these alternative, higher-resolution surface field options (for LAI, greenness/vegetation fraction, soil type, and vegetation type) in sfc_climo, then provide the user the option of which dataset they would like to use.